packages/node_modules/webex/README.md

```diff
@@ -65,8 +65,16 @@ webex.rooms.create({ title: `My First Room` }).then(room => {
 
 #### _A note on browser usage_
 
-We do not provide a built version of the SDK that includes `window.webex`.
-In-browser usage is almost the same as Node.js, but it handles the user authentication flow for you. See the [browser guide](https://webex.github.io/webex-js-sdk/guides/browsers/) for more information.
+We do provide a built, minified version of the SDK, that includes `window.webex`, which is hosted on our repo and can be used with either [unpkg](https://unpkg.com/), [jsdelivr](https://jsdelivr.com/) [gitcdn.xyz](https://gitcdn.xyz/).
+
+```html
+<!-- unpkg -->
+<script src="https://unpkg.com/webex/umd/webex.min.js"></script>
+<!-- jsdelivr -->
+<script src="https://cdn.jsdelivr.net/npm/webex/umd/webex.min.js"></script>
+<!-- gitcdn -->
+<script src="https://gitcdn.xyz/repo/webex/spark-js-sdk/master/packages/node_modules/webex/umd/webex.min.js"></script>
```

Comment:
```suggestion
<script src="https://gitcdn.xyz/repo/webex/webex-js-sdk/master/packages/node_modules/webex/umd/webex.min.js"></script>
```

Reply:
CTRL+F for other instances of `spark-js-sdk` found that `plugin-attachment-actions`'s readme still uses so i'll rename those in this PR too